### DISCUSSION TITLE
Raph Levien

### DISCUSSION THREAD

**Discussion by rljacobson** [GraphiteEditor/Graphite#1615] [Ideas/Feedback]
If you are not already familiar with his work, [Raph Levien’s blog](https://raphlinus.github.io/) is a treasure for computer graphics nerds. I recommend the following articles in particular:

 1. [Fitting cubic Bézier curves](https://raphlinus.github.io/curves/2021/03/11/bezier-fitting.html)
 1. [Parallel curves of cubic Béziers](https://raphlinus.github.io/curves/2022/09/09/parallel-beziers.html)
 2. [Simplifying Bézier paths](https://raphlinus.github.io/curves/2023/04/18/bezpath-simplify.html)

Also interesting is his work on alternatives to Béziers, like [curves based on Euler Spirals](https://raphlinus.github.io/curves/2021/02/19/parallel-curves.html). (Inkscape has support for "Spiro" paths and B-Splines, which I really like. I'd like to see more of this in drawing programs.)

Many of his algorithms are implemented in the [Kurbo](https://github.com/linebender/kurbo) Rust library.

**Comment by Keavon** [comment]
Indeed, Raph is an invaluable asset for our project. His work on Vello is especially crucial since it will be the basis for our render engine. We're often in touch about spline stuff!
